client/my-sites/themes/controller.js

```diff
@@ -130,7 +131,16 @@ export function details( context, next ) {
 		isFullScreen: true
 	} ) );
 
-	context.primary = makeElement( ThemeSheetComponent, Head, context.store, props );
 	context.secondary = null; // When we're logged in, we need to remove the sidebar.
+	//TODO: use makeElement()
+	context.primary = (
+		<ReduxProvider store={ context.store } >
+			<Head title={ props.title } isSheet>
```

Comment:
We're still using the hardcoded _Pineapple Fifteen_ title, but I think that should be hooked up in a separate PR https://github.com/Automattic/wp-calypso/blob/339c6bbb5a50d975e96c1c2c016b498044fc9107/client/my-sites/themes/head.jsx#L15